docs/tutorial.md

```diff
@@ -2,593 +2,253 @@
-
-```js
-const addition = (a, b) => {
-  return a + b
-}
-
-const test1 = addition(5, 10)
-const test2 = addition(1, 0)
-
-console.log(test1)
-console.log(test2)
+```bash
+npm init -y
 ```
 
-The `=>` is used to define a function, called the arrow operator.  
-Before the arrow operator is the parameter list, everything between the round brackets `()` are parameters, separated by a comma.  
-Parameters are variables you can give to your function so that your function can work with them.  
-Then after the arrow operator comes the function body, this is everything between the curly brackets `{}`  
-This is where you put the code of the function.  
-Now that the function is complete, we assign it to a variable to give it a name, in this case `addition`  
+This will create a `project.json` file that will store all our project's packages, so that if you want to send your code to someone else, they can easily download Mineflayer and any other dependencies you need.
 
-As you can see, this code takes the parameters `a` and `b` and adds them together.  
-Then the function will return the result.  
-When a function is defined, the code in the function body is not yet executed. To run a function you have to call it.  
-You can call a function by using the name of a function followed by round brackets. In this case `addition()`  
-However, the `addition` function requires 2 parameters. These can be passed along by putting them inside the round brackets, comma separated: `addition(1, 2)`  
-When the function is done, you can imagine that the function call is replaced by whatever the function has returned. So in this case `let test1 = addition(5, 10)` will become `let test1 = result` (You will not actually see this, but this can help you understand the concept)
+Secondly, we need to install the `mineflayer` package from the Node Package Manager (NPM). :
 
-Sometimes you will come across the following: `function addition() {}` This means the same thing, although `() => {}` is preferred. (If you really want to know why, look up 'javascript function vs arrow function')
-
-The above should output the following:
-
-```txt
-15
-1
+```bash
+npm i mineflayer
 ```
 
-#### Javascript types
-
-So far we have only worked with numbers, but Javascript can work with more variable types:
+This will put Mineflayer in the modules we have installed, and we can start using Mineflayer!
 
-- A string is a piece of text that can contain multiple characters. Strings are defined by using the quotes `''`
+### Create Your First Bot
 
-```js
-const string = 'This is a string' // string type
-```
+I would recommend starting a Minecraft server to test your bot on. A good place to host a small, simple server to host your bot on is [Aternos](https://aternos.org), although you could also host your own Minecraft server locally on `localhost` for more configuration, better ping, and access to more plugins if you're using a software like Spigot or PaperMC.
```

Comment:
I would recommend the tutorial focus on a locally hosted server for simplicity purposes. But you can still mention aternos as a good place to host a server.

Reply:
I replaced the section with

I would recommend starting a Minecraft server to test your bot on. A good place to start is to host the server [on your own computer](https://help.minecraft.net/hc/en-us/articles/360058525452-How-to-Setup-a-Minecraft-Java-Edition-Server). If you need to test your bot with plugins, you can use a software such as [PaperMC](https://papermc.io/). If you are too lazy to go through the hassle of hosting locally, you could use a free host such as [Aternos](https://aternos.org/).